python/lsst/faro/measurement/TractTableMeasurementTasks.py

```diff
@@ -144,3 +150,123 @@ def run(
                 metricName, np.mean(np.abs(result["corr"])), extras=extras
             )
         )
+
+
+class WPerpTableConfig(MeasurementTaskConfig):
+    """Class to organize the yaml configuration parameters to be passed to
+    TExTableTask when using a parquet table input. All values needed to perform
+    TExTableTask have default values set below.
+
+    Optional Input (yaml file)
+    ----------
+    Column names specified as str in yaml configuration for TeX task. These are
+    the desired column names to be passed to the calcuation. If you wish to use
+    values other than the default values specified below, add the following e.g.
+    line to the yaml file:
+
+    config.measure.raColumn = "coord_ra_new"
+    """
+
+    columns = DictField(
+        doc="""Columns required for metric calculation. Should be all columns in SourceTable contexts,
+        and columns that do not change name with band in ObjectTable contexts""",
+        keytype=str,
+        itemtype=str,
+        default={"ra": "coord_ra",
+                 "dec": "coord_dec",
+                 }
+    )
+    columnsBand = DictField(
+        doc="""Columns required for metric calculation that change with band in ObjectTable contexts""",
+        keytype=str,
+        itemtype=str,
+        default={"psfFlux": "psfFlux"
+                 }
+    )
```

Comment:
If you used the action infrastructure for this you'd get the columns needed for free and wouldn't need the config options.